cli/command/cli.go

```diff
@@ -125,25 +133,36 @@ func (cli *DockerCli) Initialize(opts *cliflags.ClientOptions) error {
 	if err != nil {
 		return err
 	}
+	cli.clientInfo = ClientInfo{
+		HasExperimental: isExperimentalCli(cli.configFile),
+	}
 	cli.initializeFromClient()
 	return nil
 }
 
+func isExperimentalCli(configfile *configfile.ConfigFile) bool {
+	if e := os.Getenv("DOCKER_EXPERIMENTAL_CLI"); e != "" {
+		value := strings.ToLower(e)
+		return value == "1" || value == "y" || value == "yes"
```

Comment:
no support for "true"?

Probably strconv.ParseBool should be used

Reply:
Thx @AkihiroSuda 🤗